src/bindings/wasm/tvgWasmLottieAnimation.cpp

```diff
@@ -118,7 +118,7 @@ struct TvgEngineMethod
     }
 
     void loadFont() {
-        Text::load("default", reinterpret_cast<const char*>(DEFAULT_FONT), DEFAULT_FONT_SIZE, "ttf", false);
+        Text::load("default", get_default_font(), DEFAULT_FONT_SIZE, "ttf", false);
```

Comment:
Please confirm if decompression can be performed globally only once across multiple instances.